bugbug/models/bugtype.py

```diff
@@ -171,7 +115,7 @@ def get_labels(self) -> tuple[dict[int, np.ndarray], list[str]]:
 
         for bug_data in bug_map.values():
             target = np.zeros(len(TYPE_LIST))
-            for type_ in bug_to_types(bug_data, bug_map):
+            for type_ in bug_features.infer_bug_types(bug_data, bug_map):
                 target[TYPE_LIST.index(type_)] = 1
```

Comment:
We could use `self.label_extractors` here instead:
```suggestion
            for i, is_type in enumerate(self.label_extractors):
                if is_type(bug_data, bug_map)
                    target[i] = 1
```

In this way, we will not need to use `bug_features.infer_bug_types()` at all. Also, we will not need `TYPE_LIST ` as well.

Reply:
Since the `bug_type_extractors` list was created in the `BugTypes` class:

https://github.com/mozilla/bugbug/blob/e5b0b2f52a8774e9a524bc97123f7afd3b9e018e/bugbug/bug_features.py#L853-L863

It can be used to generate the `label_keywords` set:

https://github.com/mozilla/bugbug/blob/e5b0b2f52a8774e9a524bc97123f7afd3b9e018e/bugbug/models/bugtype.py#L27-L36